spectrum/src/main/res/values/attrs.xml

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <attr name="spectrum_colors" format="reference" />
+    <attr name="spectrum_strokeWidth" format="dimension" />
```

Comment:
The more I think about it, I like `borderWidth` even better. The fact that it's a stroke isn't really relevant.